poll/public/handlebars/poll_studio.handlebars

```diff
@@ -0,0 +1,22 @@
+<script id="poll-form-component" type="text/html">
+{{#each items}}
+<li class="field comp-setting-entry is-set poll-{{noun}}-studio-item">
+    <div class="wrapper-comp-setting">
+        <label class="label setting-label poll-setting-label" for="{{noun}}-label-{{key}}">{{noun}}</label>
+        <input class="input setting-input" name="{{noun}}-label-{{key}}" id="{{noun}}-label-{{key}}" value="{{text}}" type="text" /><br />
+        {{#if image}}
+        <label class="label setting-label" for="{{noun}}-img-{{key}}">Image URL</label>
+        <input class="input setting-input" name="{{noun}}-img-{{key}}" id="{{noun}}-img-{{key}}" value="{{img}}" type="text" />
+        {{/if}}
+        <div class="poll-move">
+            <div class="poll-move-up">&#9650;</div>
+            <div class="poll-move-down">&#9660;</div>
+        </div>
+    </div>
+    <span class="tip setting-help">
+        {{#if image}}This must have an image URL or text, and can have both.{{/if}}
+    </span>
+    <a href="#" class="button action-button poll-delete-answer" onclick="return false;">Delete</a>
```

Comment:
This should be a ```<button>``` not an ```<a>```.  Buttons do something and links go somewhere.  If you find yourself with an (invalid) URL ref of ```#``` it usually means you should be using a button.